examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2415,6 +2415,6 @@ fn new_channel_chain(nest_count: u32) -> Result<oak::ReadHandle, Box<dyn std::er
     Ok(inner_rh)
 }
 
-oak::entrypoint!(panic_main => |_in_channel| {
+oak::entrypoint!(panic_main<()> => |_receiver| {
```

Comment:
Not sure I get your point, are you saying to skip the generic type here since it's irrelevant? The current macro requires the type to be specified, and I don't think it's worth maintaining another version of the macro without types. Or did I miss your point?

Reply:
Aah, sorry if that wasn't very clear. You are right though, I was suggesting to omit the generic type if it's not used. I think macro_rules! works a bit like a match block, so you can have multiple matchers, and I was imagining that we might have a matcher without the type specified for this case. If that doesn't work or would result in a lot of duplicate code, I'm fine to omit it.